draft-ietf-rtcweb-jsep.xml

```diff
@@ -4550,9 +4594,52 @@ placeholder
      <t>
        <figure>
          <artwork alt="answer-C1">
-        <![CDATA[
-placeholder
-         ]]>
+<![CDATA[
+v=0
+o=- 7184966905803596701 1 IN IP4 0.0.0.0
+s=-
+t=0 0
+a=ice-options:trickle
+a=group:BUNDLE a1 v1
+
+m=audio 9 UDP/TLS/RTP/SAVPF 96 0 8 97 98
+c=IN IP4 0.0.0.0
+a=mid:a1
+a=sendonly
+a=rtpmap:96 opus/48000/2
+a=rtpmap:0 PCMU/8000
+a=rtpmap:8 PCMA/8000
+a=rtpmap:97 telephone-event/8000
+a=rtpmap:98 telephone-event/48000
+a=maxptime:120
+a=extmap:1 urn:ietf:params:rtp-hdrext:sdes:mid
+a=extmap:2 urn:ietf:params:rtp-hdrext:ssrc-audio-level
+a=msid:3a0301bb-d0fa-4bf8-9ad4-86fbdf9f3aba
+       49c4f187-f00c-4dfc-9e51-dea2267f6899
+a=ice-ufrag:M4Wh
+a=ice-pwd:ogwcYFAXKMJ6OjYAEV8dy6yj
+a=fingerprint:sha-256
+              D4:7C:F0:46:10:D9:5F:90:5C:DB:DD:17:2C:38:45:BC
+              
+a=setup:active
+a=dtls-id:1
+a=rtcp-mux
+a=rtcp-rsize
+
+m=video 9 UDP/TLS/RTP/SAVPF 100 101
+c=IN IP4 0.0.0.0
+a=mid:v1
+a=sendonly
+a=rtpmap:100 VP8/90000
+a=rtpmap:101 rtx/90000
+a=fmtp:101 apt=100
+a=extmap:1 urn:ietf:params:rtp-hdrext:sdes:mid
+a=rtcp-fb:100 ccm fir
+a=rtcp-fb:100 nack
+a=rtcp-fb:100 nack pli
+a=msid:3a0301bb-d0fa-4bf8-9ad4-86fbdf9f3aba
```

Comment:
these parameters are functions of the transport more than the camera so seems like might be OK